test/meson.build

```diff
@@ -1,4 +1,21 @@
-gtest_dep = dependency('gtest', main : true, required : true)
+gtest_dep = dependency('gtest', main : true, required : false)
```

Comment:
You need to use [disabler](https://mesonbuild.com/Reference-manual_functions.html#dependency_disabler) so it will work in the `if` check.